packages/database/src/migrations/20210506052033-AddSamoaCovidPercentageVaccinatedMeasures-modifies-data.js

```diff
@@ -0,0 +1,127 @@
+'use strict';
+
+import { insertObject, generateId, codeToId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const getMeasureBuilderConfig = dataElementCode => ({
+  aggregations: [
+    {
+      type: 'FINAL_EACH_DAY',
+    },
+  ],
+  measureBuilders: {
+    numerator: {
+      measureBuilder: 'sumAllPerOrgUnit',
+      measureBuilderConfig: {
+        dataElementCodes: [dataElementCode],
+        entityAggregation: {
+          dataSourceEntityType: 'village',
+        },
+      },
+    },
+    denominator: {
+      measureBuilder: 'sumAllPerOrgUnit',
+      measureBuilderConfig: {
+        dataElementCodes: ['population_WS001'],
```

Comment:
Just wanted to note: this will sum all data for this dataElement (but it shouldn't matter as long as that is kept in mind if the population is updated)